249 Updated terms related to BIOS and firmware (#250)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

[discrete]
[[pxe]]
==== image:images/yes.png[yes] PXE (noun)

> *Description*: "PXE" is an acronym for "Pre-Boot Execution Environment". Pronounced "pixie", PXE is one of the components of the Intel Wired for Management (WfM) specification. It allows a workstation to boot from a server on a network in preference to booting the operating system on the local hard drive. PXE is a mandatory element of the WfM specification. To be considered compliant, PXE must be supported by the computer's firmware and its NIC.

Reviewer: ```suggestion
*Description*: "PXE" is an acronym for "Pre-Boot Execution Environment". Pronounced "pixie", PXE is one of the components of the Intel Wired for Management (WfM) specification. Using PXE, a workstation can boot from a server on a network rather than booting the operating system from a local hard drive. PXE is a mandatory element of the WfM specification. To be considered compliant, PXE must be supported by the computer's firmware and its NIC.
```
- Avoid "allows" or "enables" for inanimate objects
